nnfabrik/examples/mnist/trainer.py

```diff
@@ -17,6 +18,21 @@ def __init__(self, model, dataloaders, seed, epochs=5):
         self.loss_fn = nn.NLLLoss()
         self.optimizer = optim.Adam(self.model.parameters())
 
+    def main_loop(self, x, y) -> Tuple[float, float]:
```

Comment:
I believe `main_loop` is what this part of the code is typically called. At least I know it this way.

Reply:
I guess it makes sense to people that are familiar with this convention but for other people a different name might be more informative.

Reply:
Fair enough. Maybe `train_loop`? I would associate `make_step` too much with the datajoint `make`.

Reply:
I think `train_loop` is definitely better than `main_loop` because it associates the method with the `train` method. Another option would be to split the `main_loop` into `forward`, `backward` and `track_accuracy` and call these three from the `train` method but that might be overkill.

Reply:
I think I'll call it `train_loop` for the purpose of this demo.